elastic/beats
# Fix: docker compose pip dependencies

## What does this PR do?



Ti changes the version of PyYAML to 5.4.1 due to dependency conflicts with docker-compose 1.29.2. The version of PyYAML must be 6< to fix the conflict.

## Why is it important?



Several targets are broken because of this conflict.

## Checklist




## How to test this PR locally



Prerequisites

```bash
GO_VERSION=$(cat .go-version)
gvm install ${GO_VERSION}
eval $(gvm use ${GO_VERSION})

PYENV_VERSION=$(cat .python-version)
pyenv install ${PYENV_VERSION}
```

```bash
make -C beats/x-pack/heartbeat/ironbank package
```

```bash
mage -d metricbeat goIntegTest
```

```bash
mage -d metricbeat pythonIntegTest
```

```bash
mage -d x-pack/metricbeat goIntegTest
```

```bash
mage -C x-pack/metricbeat pythonIntegTest
```

```bash
make -C metricbeat check MAGEFILE_VERBOSE=1
```

```bash
make -C x-pack/metricbeat check MAGEFILE_VERBOSE=1
```

```bash
make -C metricbeat check MAGEFILE_VERBOSE=1
```

```bash
make -C x-pack/metricbeat check MAGEFILE_VERBOSE=1
```

```bash
make -C metricbeat  build unitTest MAGEFILE_VERBOSE=1 
```

```bash
make -C x-pack/metricbeat  build unitTest MAGEFILE_VERBOSE=1
```
